src/fetchconfig/fetch.js

```diff
@@ -46,12 +46,11 @@ async function fetchConfigUncached(opts) {
 // keep it cachy.
 const fetchConfigCached = cache(fetchConfigUncached, {
   hash: (fn, {
-    owner, repo, ref, _, options,
+    url, options, name,
```

Comment:
why has this changed ?

Reply:
I've updated the title. This fixes a severe caching bug. The owner, repo, and ref are not part of the options and cannot be used as a cache key.